src/SYNQLDM.m

```diff
@@ -330,6 +330,58 @@
  S @G@(MAP,"CODE","3173-2","PTT")=""
  S @G@(MAP,"CODE","2276-4","FERRITIN")=""
  S @G@(MAP,"CODE","2746-6","PH ")="" ; not a typo, it really has a space
+ ; "Leukocytes [#/volume] in Blood","26464-8",11.39 -> WBC (part of CBC)
+ S @G@(MAP,"CODE","26464-8","WBC")=""
+ ; "Erythrocytes [#/volume] in Blood","26453-1",5.6994 -> RBC (part of CBC)
+ S @G@(MAP,"CODE","26453-1","RBC")=""
+ ; "Hemoglobin","718-7",14.748 -> HGB (part of CBC)
+ S @G@(MAP,"CODE","718-7","HGB")=""
```

Comment:
duplicate

Reply:
this is not a duplicate... there is only one 26453-1 in the file.